pkg/controller/cloud/node_controller.go

```diff
@@ -138,6 +138,7 @@ func NewCloudNodeController(
 // via a goroutine
 func (cnc *CloudNodeController) Run(stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
+	klog.Info("Starting CloudNodeController")
```

Comment:
nit: suggest to remove this line, as it seems for debug only purpose.

Reply:
Prefer leave it as it is as it happens only once and we haven't found it in our log yet.